src/bundle/ui-dev/src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -244,7 +244,7 @@ const UniversalDiscoveryModule = (props) => {
             .join(',');
 
         if (!locationIds) {
-            return Promise.resolve([]);
+            return Promise.resolve(null);
```

Comment:
Note: changed to null, because when data is present we have an object with data so resolving sometimes with an object and sometimes with an empty array was confusing.